Revit_Engine/Query/DefaultDiscipline.cs

```diff
@@ -48,18 +48,23 @@ public static partial class Query
 
         /***************************************************/
 
-        [Description("Gets Default Discipline for given FilterRequest.")]
-        [Input("filterRequest", "FilterRequest")]
+        [Description("Gets Default Discipline for given Request.")]
+        [Input("request", "IRequest")]
         [Output("Discipline")]
-        public static Discipline? DefaultDiscipline(this FilterRequest filterRequest)
```

Comment:
I don't see the point of this. Why defining the method for a generic IRequest if then it works only with FilterRequest?
I'd this method needs to be changed to either to accept a new, appropriate implementation of IRequest.

It's particularly important here because the `Equalities` Dictionary property will be soon removed from the FilterRequest.